include/openssl/bn.h

```diff
@@ -64,6 +64,13 @@ extern "C" {
 # define BN_FLG_MALLOCED         0x01
 # define BN_FLG_STATIC_DATA      0x02
 
+/* Values for |top| and |bottom| in BN_rand() */
+#define BN_RAND_TOP_ANY    -1
+#define BN_RAND_TOP_ONE     0
+#define BN_RAND_TOP_TWO     1
+#define BN_RAND_BOTTOM_ANY  0
+#define BN_RAND_BOTTOM_ODD  1
+
```

Comment:
Nitpick: I would stick a newline between `BN_RAND_TOP_TWO` and `BN_RAND_BOTTOM_ANY` and maybe have separate comments for each. Also I'd suggest moving them down to below the `BN_FLG_FREE` definition. Otherwise you're breaking up the `BN_FLG_*` values in the middle which is a little odd.